template/page.phtml

```diff
@@ -6,9 +6,9 @@
 </head>
 <body>
   <?php require_once __DIR__.'/../template/menu.phtml';?>
-<main id="<?php echo $content;?>">
-  <?php require_once __DIR__.'/../template/content/'.$content.'.phtml';?>
-</main>
+  <main id="<?php echo $content; ?>">
+      <?php require_once __DIR__ . '/../template/content/' . $content . '.phtml'; ?>

```

Comment:
@ReginaBorisova 
This seems correct as well. If file is not found check the path to the file